greentest/2.7/test_ssl.py

```diff
@@ -107,17 +126,19 @@ def test_parse_cert(self):
         # note that this uses an 'unofficial' function in _ssl.c,
         # provided solely for this test, to exercise the certificate
         # parsing code
-        p = ssl._ssl._test_decode_cert(CERTFILE, False)
+        try:
+            p = ssl._ssl._test_decode_cert(CERTFILE, False)
```

Comment:
Isn't the second arg just a "verbose" flag? Can we just remove the second arg altogether?